### DISCUSSION TITLE
Cache delete mechanism

### DISCUSSION THREAD

**Discussion by liukaiyuliukaiyu** [Kong/kong#12887] [Help]
When a new kong instance is added to the original kong cluster, the cache emptying of all kong instances is triggered. What is the reason? The new instance connects to the database postgres to obtain the latest configuration without affecting the cache of other instances
![image](https://github.com/Kong/kong/assets/20014692/8d05bd48-d153-440b-9bb7-4ac282895d19)

Purge the cache when a new node joins，why???

**Comment by liukaiyuliukaiyu** [comment]
who can help me?

**Comment by chronolaw** [comment]
Could you tell us the details of kong, such as version, run options?

**Reply by liukaiyuliukaiyu** [reply]
The kong version we used is an older version, 0.10.3
serf is also being used for synchronization of cluster configuration information
The latest data is stored in the pg database, and the newly added kong instance can simply connect to pg to get the latest configuration, so why clear the cache of all other instances in the cluster

**Reply by chronolaw** [reply]
Sorry, it is too old, I think that kong has improved very much since 0.x.
I suggest you take a taste of latest 3.6 version, we do not maintain that 0.x version any more.

**Reply by liukaiyuliukaiyu** [reply]
Thank you very much for your reply. Thank you very much. The official reply is more authoritative.
I know we're on an old version, but upgrading from 0.x to 3.x was a big challenge for us. We needed to ensure the stability of our services. This is an old system, and we can only optimize based on 0.10.3.
Now with 3.6, when a new kong node joins the cluster, does it clear the cache of all kong nodes in the cluster？

**Reply by chronolaw** [reply]
I did not see purge cache code in kong 3.x, and I think that it should not happen.

The best way is setting up a new kong 3.x cluster and verify it in your own environment.
